projects/fsastorefrontlib/src/cms-components/product/product.module.ts

```diff
@@ -19,6 +24,8 @@ import { MediaModule, ListNavigationModule } from '@spartacus/storefront';
     RouterModule,
     MediaModule,
     ListNavigationModule,
+    FacetListModule,
```

Comment:
`FacetListModule ` is already imported in the `ProductFacetNavigationModule `, so you can remove it here.

Reply:
Right, removed.